DataFormats/DetId/interface/DetId.h

```diff
@@ -23,7 +23,7 @@ class DetId {
   static const int kSubdetOffset       = 25;
 
 
-  enum Detector { Tracker=1,Muon=2,Ecal=3,Hcal=4,Calo=5,Forward=6 };
+  enum Detector { Tracker=1,Muon=2,Ecal=3,Hcal=4,Calo=5,Forward=6, Trigger = 7 };
```

Comment:
Do 1), the CFC will never be build, we can commandeer that enum for our purposes.